lib/data_hygiene/publishing_api_sync_check.rb

```diff
@@ -57,6 +59,8 @@ def initialize(scope)
           Whitehall.url_maker.polymorphic_path(model)
         end
       end
+
+      add_translations_expectation
```

Comment:
Some models don't support translations, so this expectation won't need to be added at all.